Realm/RLMSyncSession.h

```diff
@@ -219,12 +219,16 @@ NS_REFINED_FOR_SWIFT;
 /// Wait for pending downloads to complete or the session to expire, and dispatch the callback onto the specified queue.
 - (BOOL)waitForDownloadCompletionOnQueue:(nullable dispatch_queue_t)queue callback:(void(^)(NSError * _Nullable))callback NS_REFINED_FOR_SWIFT;
 
+/// :nodoc:
++ (void)immediatelyHandleError:(RLMSyncErrorActionToken *)token syncManager:(RLMSyncManager *)syncManager
+    __attribute__((deprecated("The syncManager: parameter is no longer required")));
+
 /**
  Given an error action token, immediately handle the corresponding action.
- 
- @see `RLMSyncErrorClientResetError`, `RLMSyncErrorPermissionDeniedError`
+
+ @see ```RLMSyncErrorClientResetError``, ``RLMSyncErrorPermissionDeniedError``
```

Comment:
Looks like we have an extra backtick here

```suggestion
 @see ``RLMSyncErrorClientResetError``, ``RLMSyncErrorPermissionDeniedError``
```